app/scripts/modules/appengine/serverGroup/configure/wizard/cloneServerGroup.controller.ts

```diff
@@ -1,4 +1,4 @@
-import {module} from 'angular';
+import * as angular from 'angular';
```

Comment:
This seems to be the only non-spec file where you swapped the `import {module}` to `import *`.